pkgs/development/libraries/qt-5/5.15/srcs/default.nix

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
I don't think we need this file anymore. The framework name always maps to the name with json and the version is the same across.

Reply:
The version is _not_ the same for all of them, because upstream has bumped the versions differently. :stuck_out_tongue_winking_eye: